client/i18n/locales/english/intro.json

```
@@ -1568,7 +1568,12 @@
   },
   "b1-english-for-developers": {
     "title": "B1 English for Developers (Beta)",
-    "intro": ["Placeholder intro"],
+    "intro": [
+      "In this English for Developers Curriculum, you'll learn the essentials of English communication. This will follow the B1 level of the Common European Framework of Reference (CEFR). And we've focused on vocabulary that is particularly useful for developers.",
+      "It will help you strengthen your foundational skills while introducing more complex grammar and usage. You'll learn how to describe places and things, share past experiences, and confidently use tenses like the Present Perfect and Future forms. Practical communication strategies are included as well, such as managing conversations, expressing opinions, and building agreement or disagreement in discussions.",
```

```suggestion
      "It will help you strengthen your foundational skills while introducing more complex grammar and usage. You'll learn how to describe places and things, share past experiences, and confidently use tenses like Present Perfect and Future. Practical communication strategies are included as well, such as managing conversations, expressing opinions, and building agreement or disagreement in discussions.",
```
It seemed a little odd, but I'm not going to say it's wrong!

---

Thanks for that, @ojeytonwilliams! Now we just wait for the tests to pass.